pwa/src/pages/entities/[id].tsx

```diff
@@ -1,23 +1,14 @@
 import * as React from "react";
 import Layout from "../../components/common/layout";
 import AttributeTable from "../../components/attributes/attributeTable";
-import DataTable from "../../components/object_entities/dataTable";
+import {DataTable} from "../../components/object_entities/dataTable";
```

Comment:
If you create a default export in `dataTable.tsx`, this change won't be necessary. 

In `dataTable.tsx`, as the last line of the file (so **after** the final closing bracket) add: `export default DataTable;`